staging/src/k8s.io/api/testdata/v1.14.0/core.v1.Pod.after_roundtrip.json

```diff
@@ -1034,7 +1034,8 @@
         "restartCount": 1272233359,
         "image": "330",
         "imageID": "331",
-        "containerID": "332"
+        "containerID": "332",
+        "started": null
```

Comment:
we shouldn't be adding new fields on roundtrip. the `testdata/v...` changes should be dropped (the changes to `testdata/HEAD/...` look correct)